### DISCUSSION TITLE
account system

### DISCUSSION THREAD

**Discussion by pdandan** [trustwallet/wallet-core#2075] [Q&A]
If we have our own account system, should we add accounts like NEAR?
<img width="233" alt="image" src="https://user-images.githubusercontent.com/30750438/158351874-0381532b-c935-4256-a7f5-17feecfae7df.png">

**Comment by optout21** [comment]
If it is needed for interpreting SigningInput, putting together, encoding, and signing transactions, then yes.

**Reply by pdandan** [reply]
Ok,thanks!
